reference-controllers/rabbitmq-controller/src/main.rs

```diff
@@ -107,10 +142,171 @@ async fn reconcile_stateful_set(rabbitmq: &RabbitmqCluster, client: Client) -> R
             .create(&PostParams::default(), &sts)
             .await
             .map_err(Error::ReconcileStatefulSetFailed)?;
+        let gvk = GroupVersionKind::gvk("apps", "v1", "StatefulSet");
+        let api_resource = ApiResource::from_gvk(&gvk);
+        update_annotation(
```

Comment:
Why do we need this "createdAt" annotation?

Reply:
We don't need that. It is for enabling feature flags(this is a feature that enable plugins' features, but it is not an important feature). I will delete it and if we want to implement this feature, I can add it back.